### ISSUE TITLE
Feature: auto audio_codec

### ISSUE THREAD

**Issue by lafemmephile** [lafemmephile/core#45]
# Purpose
Currently the `convert` cmd has a  `-a` / `--audio-codec`  option for setting the *audio codec*. The choices are limited to: 
```python
(`aac`, `libmp3lame`, `libvorbis`)
```
Would be good to add an additional choice: `auto`

# Implementation
When `auto` is selected (it will be the default value)
```python
AUDIO_CODEC_PREFERENCES = {
  "mp4": "aac",
  "webm": "libvorbis",
  .
  .
  .
}
```

**Comment by lafemmephile** [comment]
Implemented with: 1e0c9b1f36f97137349e29d50d8289b0e2266498

**Comment by lafemmephile** [comment]
Implemented with: 1e0c9b1f36f97137349e29d50d8289b0e2266498
